src/Wrapper.js

```diff
@@ -5,6 +5,7 @@
 /* Imports -------------------------------------------------------------------*/
 
 import Inferno from 'inferno'; // eslint-disable-line
+require('./assets/scss/style.scss');
```

Comment:
I will comment the code.
Found this in a article: "Since, Webpack only understands JS. When we write a require('style.scss'), SASS loader turns it into style.css, which should then be turned into JavaScript with CSS loader, and finally embedded as styles using the style loader."